pinecone/suite_runner_test.go

```diff
@@ -10,48 +10,44 @@ import (
 	"github.com/stretchr/testify/suite"
 )
 
+// This is the entry point for all integration tests
+// This test function is picked up by go test and triggers the suite runs
+func TestRunSuites(t *testing.T) {
+	RunSuites(t)
+}
+
 func RunSuites(t *testing.T) {
 	apiKey, present := os.LookupEnv("PINECONE_API_KEY")
 	assert.True(t, present, "PINECONE_API_KEY env variable not set")
 
-	client, err := NewClient(NewClientParams{ApiKey: apiKey})
+	sourceTag := "pinecone_test_go_sdk"
```

Comment:
I've reduced the number of clients to one and applied a `sourceTag`.

Reply:
you da best